dev-libs/libxslt/libxslt-1.1.34-r1.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
A minor note:
When you update or rebase this PR next time, can you please bump libxslt to 1.1.34-r2?
We should address CVE-2021-30560, which was fixed in the r2.
See also https://bugs.gentoo.org/820722 .

Reply:
Sure, will do.

Reply:
Done. Updated libxml2 at the same time, as comments in both say that they should be updated together.